flytepropeller/pkg/controller/nodes/executor.go

```diff
@@ -1341,7 +1341,7 @@ func (c *nodeExecutor) HandleNode(ctx context.Context, dag executors.DAGStructur
 
 	if currentPhase == v1alpha1.NodePhaseTimingOut {
 		logger.Debugf(ctx, "node timing out")
-		if err := c.Abort(ctx, h, nCtx, "node timed out", false); err != nil {
+		if err := c.Abort(ctx, h, nCtx, "node timed out", true); err != nil {
```

Comment:
Does this mean that in the UI it shows up as `ABORTED` for the node phase? I believe there is a `TIMED_OUT` node phase in idl too, I'm wondering if there was a regression along the lines where we lost that. If the UI supports timed out, it might be better to send that?

Reply:
great call out. There's an issue with transforming node exec events. Updating this